ci.cue

```diff
@@ -106,5 +107,9 @@ dagger.#Plan & {
 				source: _source
 			}
 		}
+
+		"gitpod": gitpod.#Build & {
```

Comment:
This is simply to verify `gitpod` works alright, correct? (e.g. we're not using the resulting image in any way)

Reply:
Correct. I'll add a comment in the code to clarify.

Reply:
I renamed the plan to "Test" for clarity.